code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -8,6 +8,7 @@
 	heal_rate = 5
 	large = 1
 	ventcrawler = 0
+	desc = "A bulky alien. This one seems to be a jack of all trades, its only drawback being its size."

```

Comment:
inaccurate description, queens aren't the jacks of all trades they're the mothers of the nest and players should know they should stay in the nest and generate eggs

Reply:
It'll be changed.